app/dashboard/helpers.py

```diff
@@ -504,6 +504,41 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
                 except Exception as e:
                     logger.error(e)
 
+            bounty_invitees = metadata.get('invite', '')
+            print (bounty_invitees)
+            if bounty_invitees:
+                from marketing.mails import share_bounty
+                from dashboard.utils import get_bounty_invite_url
+                emails = []
+                inviter = Profile.objects.get(handle=new_bounty.bounty_owner_github_username)
+                invite_url = get_bounty_invite_url(inviter, new_bounty.id)
+                msg = "Check out this bounty that pays out" + \
```

Comment:
Are we happy with this copy here? @PixelantDesign @frankchen07 - I think the formatting here needs a space between the word `out` and the bounty value.